### PR TITLE
Update "Set up web development for Flutter" page

### LINKED ISSUE #11317: Update "Set up web development for Flutter" page
https://docs.flutter.dev/platform-integration/web/install-web

- [ ] Reduce host+platform selection options from 8 down to 3 (depending on whether your host platform is (Windows, macOS, or Linux)
- [ ] Simplify instructions and requirements.
- [ ] Install Flutter SDK, Chrome (Not required on Windows, since Edge is supported)
- [ ] Android Studio and XCode are NOT required
- [ ] Run flutter doctor, simplify these instructions
- [ ] Link to "Building a web application with Flutter" at the end
- [ ] Add "web-server" (not just "chrome") as a build target?

### PR DESCRIPTION
**Pull request by johnpryan**
This simplifies up the [install-web](https://docs.flutter.dev/platform-integration/web/install-web) instructions.

Instead of having different instructions for each platform on the [Set up web development for Flutter](https://docs.flutter.dev/platform-integration/web/install-web) page, the sidenav navigates to a single page whose instructions apply to any platform. 

cc: @kevmoo 

Resolves #11317

### CODE REVIEW

File: src/_includes/docs/install/next-steps.md [hunk 1]
```diff
@@ -24,8 +24,7 @@ consult the following resources.
 {%- endcase %}
 {%- case next-step.add-chrome %}
 {%- when 'Y' %}
-* [Add Web debugging
-  tools](/platform-integration/web/install-web/install-web-from-{{webtarget}})
+* [Add Web as a build target](/platform-integration/web/install-web/
```
Comment (sfshaza2) [inline src/_includes/docs/install/next-steps.md:27]:
```suggestion
* [Add web as a build target](/platform-integration/web/install-web/
```

File: src/_includes/docs/install/reqs/add-web.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
+Download and install [Google Chrome][] to run and debug Flutter web apps.
 
-1. Allocate a minimum of 1 GB of storage for Google Chrome.
-   Consider allocating 2 GB of storage for an optimal configuration.
-
-1. Download and install the {{include.devos}} version of [Google Chrome][]
-   to debug JavaScript code for web apps.
-
-{% if include.devos == 'Linux' %}{% render docs/install/accordions/install-chrome-from-cli.md %}{% endif %}
+Alternatively you can install [Microsoft Edge][] on Windows, or use a different
```
Comment (sfshaza2) [inline src/_includes/docs/install/reqs/add-web.md:3]:
```suggestion
Alternatively, you can install [Microsoft Edge][] on Windows, or use a different
```

File: src/_includes/docs/install/reqs/add-web.md [hunk 3]
```diff
@@ -1,10 +1,8 @@
...
+web browser by using the `flutter run -d web-server` command and navigating to
+the specified URL, but debugging is in this mode may be limited.
```
Comment (sfshaza2) [inline src/_includes/docs/install/reqs/add-web.md:5]:
```suggestion
the specified URL; however, debugging is in this mode might be limited.
```

**Review by sfshaza2 — APPROVED** [review]
OMIGOSH. This PR makes me SO HAPPY!!! I have minor edits, so I'll go ahead and approve.

**Comment by sfshaza2** [comment]
But please do incorporate them. :)

**Comment by johnpryan** [comment]
Thanks for the review, will merge once CI passes.

**Pull request merged**